tests/checkbox/checkbox-provider-tdx/units/test-plan.pxu

```diff
@@ -8,8 +8,8 @@ include:
     tdx-perf/.*
     tdx-quote/.*
     tdx-stress/.*
-#mandatory_include:
-#    com.canonical.certification::miscellanea/submission-resources
+mandatory_include:
```

Comment:
For my own understanding why is it important to include this?

Reply:
They are standardized jobs of checkbox that allow to attach the platform details on its resource (dmi, snap list, dkms, udev, ....)